widget/ui/src/components/QuoteExpanded/QuoteExpanded.tsx

```diff
@@ -0,0 +1,182 @@
+import type { LoadedPropsType, PropTypes } from './QuoteExpanded.types';
+
+import React from 'react';
+
+import { ErrorIcon, WarningIcon } from '../../icons';
+import { ChainToken } from '../ChainToken';
+import { Image } from '../common';
+import { Divider } from '../Divider';
+import { QuoteCost } from '../QuoteCost';
+import { QuoteTag } from '../QuoteTag';
+import { Tooltip } from '../Tooltip';
+import { Typography } from '../Typography';
+
+import {
+  SkeletonHeader,
+  SkeletonItemLeft,
+  SkeletonItemRight,
+  SkeletonOutput,
+} from './QuoteExpanded.Skeletons';
+import {
+  FromToken,
+  IconHighlight,
+  lastStepStyle,
+  LeftSection,
+  OutputSection,
+  RightSection,
+  RouteContainer,
+  RouteHeader,
+  StepItem,
+  Steps,
+  StyledPriceImpact,
+  SwapperContainer,
+  SwapperImage,
+  TagsContainer,
+  tokenLabelStyles,
+  VerticalLine,
+} from './QuoteExpanded.styles';
+
+const ITEM_SKELETON_COUNT = 3;
+
+export function QuoteExpanded(props: PropTypes) {
```

Comment:
I think the name of the component can be better,
For example, something like FullExpandedQuoute